web/src/main/resources/templates/routes/routes.html

```diff
@@ -66,5 +66,13 @@
     <a th:href="@{/routes/save}" class="btn btn-warning">
         Zapisz trasy
     </a></div>
+<div>
+    <form th:action="@{/routes}">
```

Comment:
Dodaj <br> to nie będzie form nachodzić
<div>
<br>
    <form th:action="@{/routes}">